build/Nuget/Microsoft.NET.Sdk.Nuget.targets

```diff
@@ -178,32 +178,32 @@ vs.localizedResources
+      <SdkSwrLines Include='%20%20file source="%24(RootOutDir)PackagesLayout\buildCrossTargeting\%(LaidOutSdk_BuildCrossTargetingFiles.Filename)%(LaidOutSdk_BuildCrossTargetingFiles.Extension)"' />      
 
       <LaidOutSdk_ToolsNet46Files Include="$(PackagesLayoutDir)tools\net46\*" />
       <SdkSwrLines Include='$(SdkSwrBlankLine)folder "InstallDir:MSBuild\Sdks\Microsoft.NET.Sdk\tools\net46"' />
-      <SdkSwrLines Include='%20%20file source="%24(OutputPath)PackagesLayout\tools\net46\%(LaidOutSdk_ToolsNet46Files.Filename)%(LaidOutSdk_ToolsNet46Files.Extension)"' />
-      <SdkSwrLines Include='$(SdkSwrBlankLine)folder "InstallDir:MSBuild\Sdks\Microsoft.NET.Sdk\tools\net46\%(SatelliteLanguage.Identity)";%20%20file source="%24(OutputPath)PackagesLayout\tools\net46\%(SatelliteLanguage.Identity)\Microsoft.NET.Build.Tasks.resources.dll"' />
+      <SdkSwrLines Include='%20%20file source="%24(RootOutDir)PackagesLayout\tools\net46\%(LaidOutSdk_ToolsNet46Files.Filename)%(LaidOutSdk_ToolsNet46Files.Extension)"' />
+      <SdkSwrLines Include='$(SdkSwrBlankLine)folder "InstallDir:MSBuild\Sdks\Microsoft.NET.Sdk\tools\net46\%(SatelliteLanguage.Identity)";%20%20file source="%24(RootOutDir)PackagesLayout\tools\net46\%(SatelliteLanguage.Identity)\Microsoft.NET.Build.Tasks.resources.dll"' />
 
       <LaidOutSdk_ToolsNet46NewtonSoftJsonFiles Include="$(PackagesLayoutDir)tools\net46\Newtonsoft.Json.6.0.4\*" />
       <SdkSwrLines Include='$(SdkSwrBlankLine)folder "InstallDir:MSBuild\Sdks\Microsoft.NET.Sdk\tools\net46\Newtonsoft.Json.6.0.4"' />
-      <SdkSwrLines Include='%20%20file source="%24(OutputPath)PackagesLayout\tools\net46\Newtonsoft.Json.6.0.4\%(LaidOutSdk_ToolsNet46NewtonSoftJsonFiles.Filename)%(LaidOutSdk_ToolsNet46NewtonSoftJsonFiles.Extension)"' />
+      <SdkSwrLines Include='%20%20file source="%24(RootOutDir)PackagesLayout\tools\net46\Newtonsoft.Json.6.0.4\%(LaidOutSdk_ToolsNet46NewtonSoftJsonFiles.Filename)%(LaidOutSdk_ToolsNet46NewtonSoftJsonFiles.Extension)"' />
 
       <LaidOutSdk_ToolsNetCoreAppFiles Include="$(PackagesLayoutDir)tools\netcoreapp1.0\*" />
       <SdkSwrLines Include='$(SdkSwrBlankLine)folder "InstallDir:MSBuild\Sdks\Microsoft.NET.Sdk\tools\netcoreapp1.0"' />
-      <SdkSwrLines Include='%20%20file source="%24(OutputPath)PackagesLayout\tools\netcoreapp1.0\%(LaidOutSdk_ToolsNetCoreAppFiles.Filename)%(LaidOutSdk_ToolsNetCoreAppFiles.Extension)"' />
-      <SdkSwrLines Include='$(SdkSwrBlankLine)folder "InstallDir:MSBuild\Sdks\Microsoft.NET.Sdk\tools\netcoreapp1.0\%(SatelliteLanguage.Identity)";%20%20file source="%24(OutputPath)PackagesLayout\tools\netcoreapp1.0\%(SatelliteLanguage.Identity)\Microsoft.NET.Build.Tasks.resources.dll"' />
+      <SdkSwrLines Include='%20%20file source="%24(RootOutDir)PackagesLayout\tools\netcoreapp1.0\%(LaidOutSdk_ToolsNetCoreAppFiles.Filename)%(LaidOutSdk_ToolsNetCoreAppFiles.Extension)"' />
```

Comment:
I think if you set AppendTargetFrameworkToOutputPath=false in Microsoft.NET.Nuget.proj, then you can revert all of these